server/controllers/refer/courseParticipationsController.ts

```diff
@@ -109,18 +109,18 @@ export default class CourseParticipationsController {
         referral.prisonNumber,
         res.locals.user.caseloads,
       )
-      const courses = await this.courseService.getCourses(req.user.token)
+      const courseNames = await this.courseService.getCourseNames(req.user.token)
 
       FormUtils.setFieldErrors(req, res, ['courseName', 'otherCourseName'])
 
       const { courseName } = courseParticipation
-      const isKnownCourse = courses.find(course => course.name === courseName)
+      const isKnownCourse = courseNames.find(name => name === courseName)
```

Comment:
`knownCourseName` might be clearer here to differentiate the two names in the iterator